galadriel/suite/state.py

```
@@ -259,31 +287,37 @@ def link_scenario(self, scenario_id:int):
-        
+
+        user_id, username = await self._get_user_info()
+        log_action(user_id, username, "linked", "suite", self.suite.name if self.suite else "", f"scenario '{scenario.name if scenario else 'unknown'}'")
+
         return rx.toast.success("scenario added!")
-    
+
 class AddSuiteState(SuiteState):
     """Handles the add-suite form submission."""
 
     form_data:dict = {}
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)